### PR TITLE
Fixed ach-login-10 and added tests

### PR DESCRIPTION
**Pull request by ArmandNM**

### CODE REVIEW

**Comment by ArmandNM** [comment]
@palcu please review

File: wouso/interface/activity/achievements.py [hunk 1]
```diff
@@ -13,18 +13,32 @@
...
-    activities = Activity.get_private_activity(player).filter(action='seen').order_by('-timestamp')[0:100]
-    today = timestamp.date()
+
+    # The maximum number of seens in the last 15 days is 24 * 15 ( we have a new 'seen' activity each hour ).
+    maximum_times_seen = 15 * 24
+
+    activities = Activity.get_private_activity(player).filter(action='seen').order_by('-timestamp')[0:maximum_times_seen]
+
+
+    print activities
```
Comment (palcu) [inline wouso/interface/activity/achievements.py]: Please remove this print and the new lines.

**Comment by palcu** [comment]
GG for fixing this.

**Comment by iulianR** [comment]
Good job!

**Pull request merged**